tests/smoke.sh

```diff
@@ -242,11 +242,18 @@ function helm_delete_stardog_release() {
 	echo "Stardog release deleted."
 }
 
+function validate_helm_chart() {
+	echo "Validating the helm chart"
+	helm lint charts/stardog >/dev/null 2>&1 || { echo >&2 "The helm chart is not valid, exiting."; exit 1; }
+	echo "Helm chart valid."
+}
+
 echo "Starting the Helm smoke tests"
 dependency_checks
 minikube_start_tunnel
 install_stardog
 helm_setup_cluster
+validate_helm_chart
```

Comment:
bring this to the top, there is no need to install stardog, or do other stuff if a helm chart is not valid. fix this and we should be good to go.